index.html

```diff
@@ -273,6 +307,11 @@
     })
   }
 
+  function submitAmount(form) {
+    calculator.amount = parseInt(form.querySelector('[name=amount]').value, 10)
+  }
+
+  calculator.init()
```

Comment:
Does it reset the calculator when the user start over? I don't get where it's done.